osxcollector/output_filters/virustotal/lookup_hashes.py

```diff
@@ -2,7 +2,7 @@
 
 # -*- coding: utf-8 -*-
 #
-# LookupDomainsFilter uses VirusTotal to lookup the values in 'md5' and add 'osxcollector_vthash' key.
+# LookupHashesFilter uses VirusTotal to lookup the values in 'sha2' and add 'osxcollector_vthash' key.
```

Comment:
ShadowServer just doesn't offer SHA2.
The goal is to use the "best" hash a service will offer.